cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -1082,7 +1082,24 @@ plainSubst s ty =
     C.TVar x       -> C.apSubst s (C.TVar x)
 
 
-cryptolURI :: [Text] -> Maybe Int -> URI
+-- | Generate a URI representing a cryptol name from a sequence of 
+--   name parts representing the fully-qualified name.  IF a "unique"
+--   value is given, this represents a dynamically bonund name in
+--   the "<interactive>" pseudo-module, and the unique value will
+--   be incorporated into the name as a fragment identifier.
+--   At least one name component must be supplied.
+--
+--   Some examples:
+--   \"Cryptol::foldl\" -> \"cryptol:/Cryptol/foldl\"
+--   \"MyModule::SubModule::name\" -> \"cryptol:/MyModule/SubModule/name\"
+--   \"<interactive>::f\" -> \"cryptol:f#1234\"
```

Comment:
Have you checked that haddock typesets these the way you expect? I'm worried that the slashes will turn some parts italic instead. These example strings probably ought to be typeset in monospace font.